go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/iotexproject/iotex-address v0.2.8
 	github.com/iotexproject/iotex-antenna-go/v2 v2.5.1
 	github.com/iotexproject/iotex-election v0.3.5-0.20210611041425-20ddf674363d
-	github.com/iotexproject/iotex-proto v0.5.15-0.20240105060115-b12b162afdde
+	github.com/iotexproject/iotex-proto v0.5.15-0.20240327004259-da069334c7da
```

Comment:
the proto PR is merged, pls update commit hash

Reply:
updated